src/components/shared/misc/ImageUploader.vue

```diff
@@ -300,9 +302,13 @@
             },
 
             containsDefaultSrc() {
-                if (this.src.indexOf(this.defaultSrc) > -1) return true
+                if (this.getDefaultSrc.length) return this.src.indexOf(this.getDefaultSrc) > -1
                 return false
             },
+
+            getDefaultSrc() {
+                return this.defaultSrc || ''
```

Comment:
So if this is null why do we want to return an empty string. Is this to fix the `this.getDefaultSrc.length` checks?